### PR TITLE
ERT equipment change and other assorted loadout changes

### PR DESCRIPTION
**Pull request by JoeHammad1844**
## About the PR

gave ERT combat boots, combat gloves, regen mesh and sutures, centcom headsets, medics get medical survival boxes and omnizine and epinephrine, engis get MV and HV cables, security get security belts and flashlights, leaders get pulse carbines and disablers, salvage get the extended capacity survival box, nukie medics get the brigmed mask and finally reinforcements get black gloves. some other loadouts get changed, like syndie footsoldiers but they're not really important, because you wont really ever see them

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: JoeHammad
- tweak: Due to ERT's lackluster performance CentCom has decided to slightly upgrade their arsenal

### CODE REVIEW

**Comment by JoeHammad1844** [comment]
no idea how to fix that

**Comment by deltanedas** [comment]
duplicate pocket1 items in something

also webedit ops real

File: Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml [hunk 1]
```diff
@@ -148,74 +149,73 @@
...
-    mask: ClothingMaskGas
-    outerClothing: ClothingOuterArmorBasic
+    mask: ClothingMaskGasSyndicate
+    outerClothing: ClothingOuterArmorBasicSlim
     ears: ClothingHeadsetAltSyndicate
     gloves: ClothingHandsGlovesCombat
     back: ClothingBackpackFilled
     shoes: ClothingShoesBootsCombatFilled
     pocket1: EnergySword
+    pocket1: EnergyShield
```
Comment (JoeHammad1844) [inline Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml:194]: darn

Comment (JoeHammad1844) [reply Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml]: the issue is resolved

File: Resources/Prototypes/Roles/Jobs/Command/centcom_official.yml [hunk 2]
```diff
@@ -15,12 +15,12 @@
...
-    shoes: ClothingShoesBootsJack
+    shoes: ClothingShoesBootsCombatFilled
     head: ClothingHeadHatCentcom
     eyes: ClothingEyesGlassesSunglasses
     gloves: ClothingHandsGlovesColorBlack
     outerClothing: ClothingOuterArmorBasic
-    id: CentcomPDA
-    ears: ClothingHeadsetAltCommand
-    pocket1: PaperOffice
+    id: CentcomIDCard
```
Comment (PuroSlavKing) [inline Resources/Prototypes/Roles/Jobs/Command/centcom_official.yml:23]: Why

**Comment by Flareguy** [comment]
Please don't give the footsoldiers a syndicate gas mask, it doesn't look nearly as good.
(the same could maybe be said about the armor vest but I haven't tested it ingame I think.)

**Comment by JoeHammad1844** [comment]
alright then

**Comment by JoeHammad1844** [comment]
just realised I need to give secoffs their pistol mags back

**Comment by JoeHammad1844** [comment]
wait, i'm stupid, I didn't remove those in the first place

**Comment by JoeHammad1844** [comment]
oh yeah I made this before sec started with guns

File: Resources/Prototypes/Roles/Jobs/Command/centcom_official.yml (hunk 2 above)
Comment (Emisse) [reply Resources/Prototypes/Roles/Jobs/Command/centcom_official.yml:23]: youll need to make a centcom pda with a legit one in it then

File: Resources/Prototypes/Roles/Jobs/Fun/emergencyresponseteam.yml [hunk 3]
```diff
@@ -17,29 +17,29 @@
...
     back: ClothingBackpackERTLeaderFilled
-    shoes: ClothingShoesBootsMag
+    shoes: ClothingShoesBootsMagAdv
     mask: ClothingMaskGasERT
     eyes: ClothingEyesGlassesSecurity
-    gloves: ClothingHandsGlovesColorBlack
+    gloves: ClothingHandsGlovesCombat
     outerClothing: ClothingOuterHardsuitERTLeader
-    suitstorage: OxygenTankFilled
+    suitstorage: AirTankFilled
```
Comment (Emisse) [inline Resources/Prototypes/Roles/Jobs/Fun/emergencyresponseteam.yml:40]: most players are not gonna understand how an air tank works and accidentally suffocate themselves, use an oxy tank

File: Resources/Prototypes/Roles/Jobs/Fun/emergencyresponseteam.yml [hunk 4]
```diff
@@ -80,11 +80,11 @@
     shoes: ClothingShoesBootsMagAdv
     mask: ClothingMaskGasERT
     eyes: ClothingEyesGlassesMeson
-    gloves: ClothingHandsGlovesColorYellow
+    gloves: ClothingHandsGlovesCombat
     outerClothing: ClothingOuterHardsuitERTEngineer
-    suitstorage: OxygenTankFilled
+    suitstorage: AirTankFilled
```
Comment (Emisse) [inline Resources/Prototypes/Roles/Jobs/Fun/emergencyresponseteam.yml:85]: ditto

File: Resources/Prototypes/Roles/Jobs/Fun/emergencyresponseteam.yml [hunk 5]
```diff
@@ -125,13 +126,14 @@
     shoes: ClothingShoesBootsMag
     mask: ClothingMaskGasERT
     eyes: ClothingEyesGlassesSecurity
-    gloves: ClothingHandsGlovesColorBlack
+    gloves: ClothingHandsGlovesCombat
     outerClothing: ClothingOuterHardsuitERTSecurity
-    suitstorage: OxygenTankFilled
+    suitstorage: AirTankFilled
```
Comment (Emisse) [inline Resources/Prototypes/Roles/Jobs/Fun/emergencyresponseteam.yml:131]: ditto

File: Resources/Prototypes/Roles/Jobs/Fun/emergencyresponseteam.yml [hunk 6]
```diff
@@ -171,11 +173,11 @@
     shoes: ClothingShoesBootsMag
     mask: ClothingMaskGasERT
     eyes: ClothingEyesHudMedical
-    gloves: ClothingHandsGlovesColorBlack
+    gloves: ClothingHandsGlovesCombat
     outerClothing: ClothingOuterHardsuitERTMedical
-    suitstorage: OxygenTankFilled
+    suitstorage: AirTankFilled
```
Comment (Emisse) [inline Resources/Prototypes/Roles/Jobs/Fun/emergencyresponseteam.yml:178]: ditto

File: Resources/Prototypes/Roles/Jobs/Fun/emergencyresponseteam.yml [hunk 7]
```diff
@@ -217,8 +219,8 @@
     mask: ClothingMaskBreath
     gloves: ClothingHandsGlovesColorBlack
     outerClothing: ClothingOuterHardsuitERTJanitor
-    suitstorage: OxygenTankFilled
+    suitstorage: AirTankFilled
```
Comment (Emisse) [inline Resources/Prototypes/Roles/Jobs/Fun/emergencyresponseteam.yml:222]: ditto

File: Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml [hunk 8]
```diff
@@ -42,10 +42,10 @@
     back: ClothingBackpackDuffelSyndicateAmmo
     mask: ClothingMaskGasDeathSquad
     eyes: ClothingEyesGlassesSecurity
-    ears: ClothingHeadsetAltCommand
+    ears: ClothingHeadsetAltCentCom
     gloves: ClothingHandsGlovesCombat
     outerClothing: ClothingOuterHardsuitDeathsquad
-    suitstorage: YellowOxygenTankFilled
+    suitstorage: AirTankFilled
```
Comment (Emisse) [inline Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml:48]: ditto

File: Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml [hunk 9]
```diff
@@ -240,14 +240,14 @@
...
-    ears: ClothingHeadsetAltCommand
+    ears: ClothingHeadsetAltCentCom
     gloves: ClothingHandsGlovesCombat
     outerClothing: ClothingOuterHardsuitCBURN
     shoes: ClothingShoesBootsCombatFilled
     id: CBURNPDA
     pocket1: RadioHandheld
     pocket2: WeaponLaserGun
-    suitstorage: YellowOxygenTankFilled
+    suitstorage: AirTankFilled
```
Comment (Emisse) [inline Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml:250]: ditto

**Review by Emisse — CHANGES_REQUESTED** [review]

**Comment by JoeHammad1844** [comment]
I think it's ready now

**Pull request merged**